std/experimental/allocator/package.d

```diff
@@ -605,6 +605,8 @@ unittest
     assert(a == [0, 0, 0]);
 }
 
+import std.experimental.allocator.gc_allocator: isGCSafeAllocator;
```

Comment:
> No, I mean you still can't have a global selective import because it's still broken.

It works on my machine and auto tester is passing so far - does it only fail on specific platforms?